rllib/policy/policy.py

```diff
@@ -451,6 +454,34 @@ def learn_on_batch(self, samples: SampleBatch) -> Dict[str, TensorType]:
         self.apply_gradients(grads)
         return grad_info
 
+    @ExperimentalAPI
+    def learn_on_batch_from_replay_buffer(
+            self, replay_actor: ActorHandle,
+            policy_id: PolicyID) -> Dict[str, TensorType]:
+        """Samples a batch from given replay actor and performs an update.
+
+        Args:
+            replay_actor: The replay buffer actor to sample from.
+            policy_id: The ID of this policy.
+
+        Returns:
+            Dictionary of extra metadata from `compute_gradients()`.
+        """
+        # Sample a batch from the given replay actor.
+        # For better performance, make sure the replay actor is co-located
+        #  with this policy (on the same node).
```

Comment:
maybe this comment needs to be updated?
where are we making sure the replay_actor is co-located with the policy?

Reply:
The driver (execution plan/Trainer.setup) needs to make sure of that. But it's not hard-required, just better for performance as we don't have to send the batch across the wire, then.

Reply:
I see. can. you update the comment a little bit and say:

For better performance, trainer will try to schedule replay actors co-located with this policy

something like that? thanks.

Reply:
I'm not sure this comment should be here (we don't know for sure what the Trainer will do). The policy has no influence on its Trainer. We should simply clarify that it would be better, if they WERE on the same node, but it's not a hard requirement. And that the Trainer (that creates buffer and policy) needs to take care of this, not the policy itself.

I'll fix.